tests/integration/synapseclient/test_tables.py

```diff
@@ -173,6 +174,59 @@ def test_rowset_tables(syn, project):
     assert len(row_reference_set1['rows']) == 4
 
 
+def test_materialized_view(syn, project):
+    """Test creation of materialized view"""
+    # Define schema
+    cols = [Column(name='Name', columnType='STRING'),
+            Column(name='Born', columnType='INTEGER'),
+            Column(name='Hipness', columnType='DOUBLE'),
+            Column(name='Living', columnType='BOOLEAN')]
+
+    schema = Schema(name='Jazz Guys', columns=cols, parent=project)
+
+    data = [["John Coltrane",  1926, 8.65, False],
+            ["Miles Davis",    1926, 9.87, False],
+            ["Bill Evans",     1929, 7.65, False],
+            ["Paul Chambers",  1935, 5.14, False],
+            ["Jimmy Cobb",     1929, 5.78, True],
+            ["Scott LaFaro",   1936, 4.21, False],
+            ["Sonny Rollins",  1930, 8.99, True],
+            ["Kenny Burrel",   1931, 4.37, True]]
```

Comment:
These tests aren't going to age well. haha.

Reply:
Ya... Any suggestions?

Reply:
It was a bad pun on the "living" param and whether the tests would age well. I love the jazz tests in the client.